examples/model_selection/plot_roc.py

```diff
@@ -19,44 +19,32 @@
 -------------------
```

Comment:
Maybe we don't need this `Multiclass settings` title

Reply:
I agree. Remove this heading.